packages/commonwealth/client/scripts/views/components/feed.tsx

```diff
@@ -23,11 +24,18 @@ import {
 import useUserStore from 'state/ui/user';
 import Permissions from 'utils/Permissions';
 import { z } from 'zod';
+import ThreadPreviewModal from '../modals/ThreadPreviewModal';
 import { ThreadCard } from '../pages/discussions/ThreadCard';
+import { CWModal } from './component_kit/new_designs/CWModal';
 
 const DEFAULT_COUNT = 10;
 
-const FeedThread = ({ thread }: { thread: Thread }) => {
+type FeedThreadProps = {
+  thread: Thread;
+  onClick: () => void;
+};
+
+const FeedThread: React.FC<FeedThreadProps> = ({ thread, onClick }) => {
```

Comment:
as I mentioned previously, we dont use FC in the project so please keep the consistency